pkg/reconciler/route/table_test.go

```diff
@@ -130,29 +130,23 @@ func TestReconcile(t *testing.T) {
 	}, {
 		Name: "simple route becomes ready, ingress unknown",
 		Objects: []runtime.Object{
-			Route("default", "becomes-ready", WithConfigTarget("config"), WithRouteUID("12-34"),
+			Route("default", "becomes-ready", WithConfigTarget("ing-unk"), WithRouteUID("12-34"),
```

Comment:
can we just use `unknown` instead of `unk` ?

Reply:
sure :) more typing though